From 90220d41f65910b108c901977eefe78fc1edecc5 Mon Sep 17 00:00:00 2001
From: Nicola Lunghi <nicola.lunghi@jci.com>
Date: Mon, 17 Jun 2019 13:59:26 +0100
Subject: [PATCH] Enable HotSpot builds on 5.x Linux kernels

8217753: Enable HotSpot builds on 5.x Linux kernels
Summary: Remove Linux kernel version check as very unlikely a kernel older than 2.4 will be used.
Reviewed-by: erikj

Upstream-Status: Backport (http://hg.openjdk.java.net/jdk8u/jdk8u/hotspot/rev/5af8ec63c21c)
---
 hotspot/make/linux/Makefile | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/hotspot/make/linux/Makefile b/hotspot/make/linux/Makefile
index 17c14097..e8f20104 100644
--- a/hotspot/make/linux/Makefile
+++ b/hotspot/make/linux/Makefile
@@ -227,20 +227,7 @@ all:
 	@echo "  $(TARGETS_SHARK)"
 	@echo "  $(TARGETS_MINIMAL1)"
 
-checks: check_os_version check_j2se_version
-
-# We do not want people accidentally building on old systems (e.g. Linux 2.2.x,
-# Solaris 2.5.1, 2.6).
-# Disable this check by setting DISABLE_HOTSPOT_OS_VERSION_CHECK=ok.
-
-SUPPORTED_OS_VERSION = 2.4% 2.5% 2.6% 3% 4%
-OS_VERSION := $(shell uname -r)
-EMPTY_IF_NOT_SUPPORTED = $(filter $(SUPPORTED_OS_VERSION),$(OS_VERSION))
-
-check_os_version:
-ifeq ($(DISABLE_HOTSPOT_OS_VERSION_CHECK)$(EMPTY_IF_NOT_SUPPORTED),)
-	$(QUIETLY) >&2 echo "*** This OS is not supported:" `uname -a`; exit 1;
-endif
+checks: check_j2se_version
 
 # jvmti.make requires XSLT (J2SE 1.4.x or newer):
 XSLT_CHECK	= $(REMOTE) $(RUN.JAVAP) javax.xml.transform.TransformerFactory
-- 
2.20.1

